### PR TITLE
Ajax request [Not Ready To Merge ]

### PR DESCRIPTION
**Pull request by aayusharora**

### CODE REVIEW

File: src/generator/www/js/main.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+ var $email =$('#email');
+ var $name  =$('#name'); 
+ var $url  =$('#url');
+
+  $('#btn').click(function() { 
+  	
+  var form = document.querySelector('form');
+  var formData = new FormData(form);
+  var file_data = $('input[type="file"]')[0].files;
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Identifier 'file_data' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710786219/issues/source?bid=3233857&fileBranchId=3344140#l11)

File: src/generator/www/js/main.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    
+  }
+  $.ajax({
+    type: 'POST',
+    url:  ' ',
+    data: obj,
+    processData: false,
+    contentType: false,
+    
+    success :function(data){
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ['data' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710786219/issues/source?bid=3233857&fileBranchId=3344140#l33)

File: src/generator/www/js/main.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  formData.append( 'trackfile', $( '#trackfile' )[0].files[0] );
+  formData.append( 'locationfile', $( '#locationfile' )[0].files[0] );
+
+  var obj={
+    name:$name.val(),
+    email:$email.val(),
+    url :$url.val(),
+    data:obj
+    
+  }
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710786219/issues/source?bid=3233857&fileBranchId=3344140#l25)

File: src/generator/www/js/main.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    url:  ' ',
+    data: obj,
+    processData: false,
+    contentType: false,
+    
+    success :function(data){
+       
+  })
+
+});
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710786219/issues/source?bid=3233857&fileBranchId=3344140#l37)

Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710786219/issues/source?bid=3233857&fileBranchId=3344140#l37)

File: src/generator/www/js/main.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+ var $email =$('#email');
+ var $name  =$('#name'); 
+ var $url  =$('#url');
+
+  $('#btn').click(function() { 
+  	
+  var form = document.querySelector('form');
+  var formData = new FormData(form);
+  var file_data = $('input[type="file"]')[0].files;
+  console.log(file_data);
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Identifier 'file_data' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710786219/issues/source?bid=3233857&fileBranchId=3344140#l12)

File: src/generator/www/js/main.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  $.ajax({
+    type: 'POST',
+    url:  ' ',
+    data: obj,
+    processData: false,
+    contentType: false,
+    
+    success :function(data){
+       
+  });
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710818670/issues/source?bid=3233857&fileBranchId=3344140#l33)

File: src/generator/www/js/main.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    
+  }
+  $.ajax({
+    type: 'POST',
+    url:  ' ',
+    data: obj,
+    processData: false,
+    contentType: false,
+    
+    success :function(data){
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ['data' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710818670/issues/source?bid=3233857&fileBranchId=3344140#l31)

File: src/generator/www/js/main.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+  formData.append( 'trackfile', $( '#trackfile' )[0].files[0] );
+  formData.append( 'locationfile', $( '#locationfile' )[0].files[0] );
+
+  var obj={
+    name:$name.val(),
+    email:$email.val(),
+    url :$url.val(),
+    data:obj
+    
+  }
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710818670/issues/source?bid=3233857&fileBranchId=3344140#l23)

**Comment by aayusharora** [comment]
@championswimmer , fill the url ..with your link and check the response , I will do the changes and then final it .

File: src/generator/www/js/main.js [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var formData = new FormData(form);
+ 
+  formData.append( 'sessionfile', $( '#sessionfile' )[0].files[0] );
+  formData.append( 'speakerfile', $( '#speakerfile' )[0].files[0] );
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l13)

File: src/generator/www/js/main.js [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var formData = new FormData(form);
+ 
+  formData.append( 'sessionfile', $( '#sessionfile' )[0].files[0] );
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l12)

File: src/generator/www/js/main.js [hunk 11]
```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l9)

File: src/generator/www/js/main.js [hunk 12]
```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() {
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l7)

File: src/generator/www/js/main.js (hunk 9 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l13)

File: src/generator/www/js/main.js [hunk 13]
```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var formData = new FormData(form);
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Unexpected var, use let or const instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l10)

File: src/generator/www/js/main.js [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+ 
+  formData.append( 'sessionfile', $( '#sessionfile' )[0].files[0] );
+  formData.append( 'speakerfile', $( '#speakerfile' )[0].files[0] );
+  formData.append( 'trackfile', $( '#trackfile' )[0].files[0] );
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [There should be no spaces inside this paren.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l14)

File: src/generator/www/js/main.js [hunk 15]
```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Identifier '$url' does not match the pattern '^[a-z]+([A-Z][a-z]+)*$'.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l5)

File: src/generator/www/js/main.js (hunk 9 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [There should be no spaces inside this paren.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l13)

File: src/generator/www/js/main.js (hunk 15 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Unexpected var, use let or const instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l5)

File: src/generator/www/js/main.js [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+  formData.append( 'trackfile', $( '#trackfile' )[0].files[0] );
+  formData.append( 'locationfile', $( '#locationfile' )[0].files[0] );
+
+  var obj={
+    name:$name.val(),
+    email:$email.val(),
+    url :$url.val(),
+    files:formData
+    
+  }
```
Comment (championswimmer) [inline src/generator/www/js/main.js]: @aayusharora So basically in the POST request there will be a multi-part form
where i'll get name, email and url in post params ? 
And how'll I retrieve the files ?

File: src/generator/www/js/main.js [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+
+  var form = document.querySelector('form');
+  var fd = new FormData(form);
+
+  var files ={
+    'sessionfile':  $( '#sessionfile' )[0].files[0],
+    'speakerfile':$( '#speakerfile' )[0].files[0],
+    'trackfile':$( '#trackfile' )[0].files[0] ,
+    'locationfile':$( '#locationfile' )[0].files[0]
+  }
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l17)

File: src/generator/www/js/main.js [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+
+  $.ajax({
+    type: 'POST',
+    url:  ' ',
+    data: obj,
+    processData: false,
+    contentType: false,
+    
+    success :function(data){
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ['data' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l38)

File: src/generator/www/js/main.js [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var fd = new FormData(form);
+
+  var files ={
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l12)

File: src/generator/www/js/main.js [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+    'sessionfile':  $( '#sessionfile' )[0].files[0],
+    'speakerfile':$( '#speakerfile' )[0].files[0],
+    'trackfile':$( '#trackfile' )[0].files[0] ,
+    'locationfile':$( '#locationfile' )[0].files[0]
+  }
+  
+  for (var key in files) {
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l19)

File: src/generator/www/js/main.js [hunk 21]
```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l9)

File: src/generator/www/js/main.js [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+  var fd = new FormData(form);
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l10)

File: src/generator/www/js/main.js [hunk 23]
```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() {
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l7)

File: src/generator/www/js/main.js [hunk 24]
```diff
@@ -0,0 +1,51 @@
...
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var fd = new FormData(form);
+
+  var files ={
+    'sessionfile':  $( '#sessionfile' )[0].files[0],
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l13)

File: src/generator/www/js/main.js [hunk 25]
```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Identifier '$url' does not match the pattern '^[a-z]+([A-Z][a-z]+)*$'.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l5)

Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Unexpected var, use let or const instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l5)

File: src/generator/www/js/main.js [hunk 26]
```diff
@@ -0,0 +1,52 @@
...
+
+  var form = document.querySelector("form");
+  var fd = new FormData(form);
+
+  var files ={
+    "sessionfile":  $( "#sessionfile" )[0].files[0],
+    "speakerfile":$( "#speakerfile" )[0].files[0],
+    "trackfile":$( "#trackfile" )[0].files[0] ,
+    "locationfile":$( "#locationfile" )[0].files[0]
+  }
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l17)

File: src/generator/www/js/main.js [hunk 27]
```diff
@@ -0,0 +1,52 @@
...
+  };
+
+  $.ajax({
+    type: "POST",
+    url:  " ",
+    data: obj,
+    processData: false,
+    contentType: false,
+    
+    success :function(data){
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ['data' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l39)

File: src/generator/www/js/main.js [hunk 28]
```diff
@@ -0,0 +1,52 @@
...
+  var fd = new FormData(form);
+
+  var files ={
+    "sessionfile":  $( "#sessionfile" )[0].files[0],
+    "speakerfile":$( "#speakerfile" )[0].files[0],
+    "trackfile":$( "#trackfile" )[0].files[0] ,
+    "locationfile":$( "#locationfile" )[0].files[0]
+  }
+  
+  for (var key in files) {
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l19)

File: src/generator/www/js/main.js [hunk 29]
```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+
+  let $email =$("#email");
+  let $name  =$("#name"); 
+  let $url  =$("#url");
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Combine this with the previous 'let' statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l5)

File: src/generator/www/js/main.js [hunk 30]
```diff
@@ -0,0 +1,52 @@
...
+
+  $("#btn").click(function() { 
+
+  var form = document.querySelector("form");
+  var fd = new FormData(form);
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l10)

File: src/generator/www/js/main.js [hunk 31]
```diff
@@ -0,0 +1,52 @@
...
+
+  var files ={
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l12)

File: src/generator/www/js/main.js [hunk 32]
```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+
+  let $email =$("#email");
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [`$email` is never modified, use `const` instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l3)

File: src/generator/www/js/main.js [hunk 33]
```diff
@@ -0,0 +1,52 @@
...
+  let $name  =$("#name");
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l4)

File: src/generator/www/js/main.js [hunk 34]
```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Use the function form of "use strict".](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l1)

File: src/generator/www/js/main.js (hunk 31 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l12)

File: src/generator/www/js/main.js (hunk 27 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ['data' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l39)

File: src/generator/www/js/main.js (hunk 26 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l17)

File: src/generator/www/js/main.js (hunk 28 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l19)

File: src/generator/www/js/main.js (hunk 29 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Combine this with the previous 'let' statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l5)

File: src/generator/www/js/main.js (hunk 31 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l12)

File: src/generator/www/js/main.js (hunk 30 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l10)

File: src/generator/www/js/main.js (hunk 32 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [`$email` is never modified, use `const` instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l3)

File: src/generator/www/js/main.js (hunk 33 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l4)

File: src/generator/www/js/main.js (hunk 34 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Use the function form of "use strict".](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l1)

File: src/generator/www/js/main.js (hunk 31 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l12)

**Comment by championswimmer** [comment]
You still need to resolve some conflicts :P

**Comment by aayusharora** [comment]
@championswimmer  , Solved :)

**Pull request merged**